### PR TITLE
Add use case in DG

### PR DESCRIPTION
**Pull request by revdrag**
Add `access a specific student’s record through a filter` use case.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -665,6 +665,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ---
 
+**Use case: Access a student's record through a filter**
+
+**MSS**
+
+1. User request to access a student's record using their name, class and category to change.
```
Comment (seanpzk) [inline docs/DeveloperGuide.md]: Slight grammar mistake (request to requests) here that you may have overlooked! 

Perhaps could make it more general and concise, black box-ing the process (the inputs needed):
"User requests to access a specific category of a student's record."

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -665,6 +665,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+* 1a. Student name, class name or category is not specified.
+  * 1a1. EduTrack informs that student name, class name or category is missing.
+
+    Use case ends.
+* 1b. EduTrack detects that student name, class name or category does not exist.
+  * 1b1. EduTrack informs the user that the student name, class name or category does not exist.
+  * 1b2. EduTrack terminates the request.
+
+    Use case ends.
+
```
Comment (seanpzk) [inline docs/DeveloperGuide.md:689]: I like the extensions given, maybe you could write them in similar style. ie 1a. starts with "EduTrack detecting missing inputs" just like 1b. It also follows the format of starting with either the Actor or the System :)

**Review by seanpzk — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (revdrag) [reply docs/DeveloperGuide.md]: Thanks for pointing that out

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (revdrag) [reply docs/DeveloperGuide.md:689]: I have changed the wording of the use case to better reflect the requirements of a use case. Let me know if there is anything else that looks out of place 😄

**Review by seanpzk — APPROVED** [review]
LGTM!

**Pull request closed without merging**